### ISSUE TITLE
Default GDNative WebRTC implementation not defined. [c#]

### ISSUE THREAD

**Issue by ASavchenkov** [godotengine/webrtc-native#16]
I'm following the [WebRTC tutorial](https://docs.godotengine.org/en/stable/tutorials/networking/webrtc.html) and converting things to c#., and got the above error on build (twice for some reason).

Latest GDNative plugin (0.3) downloaded and enabled.

I'm pretty new to Godot and my c# is rusty, but I suspect this has little to do with my code. If it does, It's attached as a txt file.

[Player.txt](https://github.com/godotengine/webrtc-native/files/4543536/Player.txt)

**Comment by ASavchenkov** [comment]
Just tried to reproduce and could not. Perhaps a restart was necessary. Should have thought of it sooner. I'll close this.

**Comment by juanitogan** [comment]
Just adding a note in case it helps others who come here.

I was getting this error during run from the editor (GDScript, not C#, in my case).  I noticed the GDNative plugin was Disabled in the Project Settings.  Enabling it fixed the call to `WebRTCPeerConnection.new()` in one place in the code but, weirdly, not the other call in nearly identical code.  Restarting did not help.  Furthermore, restarting always set the plugin back to Disabled.

Removing the `webrtc` folder from the project, restarting, and re-adding it (identical files, I checked) fixed all issues.  ¯\_(ツ)_/¯
